lib/LanguageServerCodeTransform/CodeAction/ImportNameProvider.php

```diff
@@ -87,10 +96,13 @@ private function diagnosticsFromUnresolvedName(NameWithByteOffset $unresolvedNam
             )
         );
 
-        $candidates = iterator_to_array($this->finder->candidatesForUnresolvedName($unresolvedName));
+        if ($hasCandidates === null) {
+            $hasCandidates = $this->finder->candidatesForUnresolvedName($unresolvedName)->current() !== null;
+        }
 
-        if (count($candidates) === 0) {
+        if ($hasCandidates) {
```

Comment:
Should this be `false === $hasCandidates`? If it has candidates then it can be imported, this expression leads to "clss doesn't exist"

Reply:
It would seem that I really did a half-arsed job. You are correct.